src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div id="business-type-confirm">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0 checkbox-gp-text">{{ label }}</p>
+        <div id="business-check-div">
+          <v-checkbox
+            class="mt-0"
+            v-model="checked"
+            hide-details
+            :label="text"
+          >
+          </v-checkbox>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class BusinessTypeConfirm extends Vue {
+  /** The registration business number. */
+  @Prop({ required: true })
+  readonly businessTypeConfirm!: boolean
+
+  /** Whether to display Change features. */
+  @Prop({ default: false })
+  readonly hasBusinessTypeChecked!: boolean;
+
+  /** Whether the business typ is SP or GP */
+  @Prop({ default: false })
+  readonly isTypePartnership!: boolean;
+
+  // Local variables
+  protected checked = false
+  protected label = 'General Partnership'
+  protected labelSP = 'BC Sole Proprietorship / Doing Business As (DBA) Registration'
+  protected text = `I acknowledge that a General Partnership cannot be \
+    changed into a Sole Proprietorship (including DBA). If this is \
+    necessary, a new Name Request Number and Statement of Registration \
+    (along with associated fees) will be required.`
+  protected textSP = `I acknowledge that a Sole Proprietorship (including DBA) \
+    cannot be changed into a General Partnership. If this is necessary, a new \
+    Name Request Number and Statement of Registration (along with associated fees) \
+    will be required.`
+
+  /** Called when component is mounted. */
+  protected mounted (): void {
+    // init model variable + validate
+    this.checked = this.businessTypeConfirm
+    this.label = this.isTypePartnership ? this.label : this.labelSP
+    this.text = this.isTypePartnership ? this.text : this.textSP
+    this.emitValid()
+  }
```

Comment:
According to the latest [design](https://projects.invisionapp.com/share/3511UPXPMT2P#/screens/460287155_Step_1-2_SP_-_DBA_Errors), I included added SP for this component and used conditional statements to render the label and text message.